src/backend/tests/app/test_flask_app.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import expungeservice
+from flask import g
+
+
+class TestFlaskApp(unittest.TestCase):
+
+    def setUp(self):
+        self.app = expungeservice.app.create_app('development')
+        self.test_client = self.app.test_client()
+
+
+    def test_get_database_connection(self):
+
+        with self.app.app_context():
+
+            expungeservice.request.before()
+
+            assert g.database
+
+            query = 'SELECT * FROM users;'
+            g.database.cursor.execute(query, ())
+            rows = g.database.cursor.fetchall()
+            assert rows or rows == []
```

Comment:
I'd be tempted to change this to just `assert rows == []`. The reasoning behind this is that before every test case is ran the database should be clean. Currently `assert rows` is falsey, and it evaluates `rows == []`. In the future if the database does not get cleaned before the test run and there are users in the table then `assert rows` will evaluate to true, however if its changed to `assert rows == []` then this will be a good indicator to let us know that the database is not being cleaned before each test run.  What do you think?

Reply:
The reason I wanted to allow it to pass on a non-empty result is that a user might have manually put in some rows that don't need to be cleaned up by/for pytest. I've been doing this when poking at the database interactively. This test doesn't distinguish the result, only that a generic hit to the database worked.

Reply:
Understood. Ideally we would have a separate test database, development database, and production database. I've worked with apps that have tests that depend on data already being in the test database, and it's an OK approach to take, but it's been my experience that it causes unexpected issues. Just something to keep in mind.